### ISSUE TITLE
How to make a request and get magnet links of the best results

### ISSUE THREAD

**Issue by Casvt** [Sonarr/Sonarr#4357]
Hey there. 

Eventhough it sounds pretty easy, I'm not able to find the anwser anywhere.

All I want is to make an api request to the jackett api and get an result. I want to insert a search term (Modern Family S01E01 Pilot) in the api request, and get a result of all my indexers. Just a list of all the results of all my indexers. With the best result on top. And (directly or indirectly by making a second request) get the magnet link of that torrent.

Basically a search engine for torrents. That's all I want. Sonarr and Radarr probably do this, so that would mean that I could do that to.

Hopefully you can help!

**Comment by bakerboy448** [comment]
Closing.
Template not followed
not relevant to sonarr
